app/webpack/taxa/show/components/status_tab.jsx

```diff
@@ -141,6 +141,17 @@ const StatusTab = ( { statuses, listedTaxa, listedTaxaCount } ) => {
                       text={status.description}
                     />
                   ) }
+                  { status.taxon_id && status.taxon_name && status.taxon_id !== taxonId && (
+                      <div
+                          className="text-muted"
+                          dangerouslySetInnerHTML={{
+                            __html: I18n.t( "status_applied_from_higher_level_taxon_html", {
+                              url: `/taxa/${status.taxon_id}`,
```

Comment:
If you end up passing in a taxon object instead of the taxon ID, you could use `urlForTaxon` in app/webpack/taxa/shared/util.js.